### ISSUE TITLE
Little exploit issue

### ISSUE THREAD

**Issue by hipopcorn** [cSploit/android#321]
When you edit options of an exploit, for example IP 192.168.1.10, and you start searching exploit in another IP, the edit options will be the other IP, for example, if i edit options of an exploit on the 192.168.1.88 it will still 192.168.1.10, its a little detail that will make the exploits doesn't work for that ip.
Another issue, is the exploit options set to one port and the payload listen using another port, than you need to always edit (only in a few exploits)

**Comment by tux-mind** [comment]
Hi @Haxin-hipopcorn , thanks for the report.

can you explain the steps to reproduce this ?

  - search for exploit on target A
  - switch to target B, which has some exploit
  - edit target B exploit option 

right ?

**Comment by Haxin-hipopcorn** [comment]
@tux-mind search for target A exploit, edit it to use a certain IP
Search for target B the target A IP stays in B.
It will be better if we have a condition, when you edit the options the IP address of the target is automatically set in RHOST, like armitage does

**Comment by tux-mind** [comment]
yeah, the msf_restyling branch will provide many new feature, one of them is a shared global datastore for all msf modules.

thanks for the report, will fix when I'll finish up the msf_restyling branch.

priority unchanged BTW, isn't a low prio IMHO but will take a while to fix.

**Comment by Haxin-hipopcorn** [comment]
@tux-mind okay, thanks for helping with this bug.
